chainer/links/connection/bilinear.py

```diff
@@ -24,13 +23,17 @@ class Bilinear(link.Link):
             ``(left_size, right_size, out_size)``. If ``None``,
             :math:`W` is initialized by centered Gaussian distribution properly
             scaled according to the dimension of inputs and outputs.
+            May also be a callable that takes ``~numpy.ndarray`` or
```

Comment:
`~` removes the module name. In this context the name is important. So, you need to use `numpy.ndarray`.